### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
Interactive

### CODE REVIEW

**Comment by mukhammadrizooff** [comment]
# I tried running the ESlint to check for the errors in my local machine and I found none which is being shown by GitHub actions.
I need help with how to go about this problem kindly.

**Comment by mukhammadrizooff** [comment]
## Everything is working well

### If you will try the Live demo link by clicking view deployment in the PR section it shows the README.md file. I would ask you to go to this [Page](https://github.com/mukhammadrizooff/To-Do-List/tree/interactive-list) and try it out from the link in my `README.md` file. It is fully functional.

**Comment by mukhammadrizooff** [comment]
# Thank you for your great help @Rustamxon7

File: src/display.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+import {
+  addNewEvents,
+  deleteAllElements,
+  deleteListEvents,
+  dragDropEvents,
+  editListEvents,
+  listCompleteEvents,
+} from './elemevent.js';
+import List from './root.js';
```
Comment (rustam-tolipov) [inline src/display.js:11]: ## WOW, GOOD JOB 🎊.👏 
You have added drag and  drop functionality correctly and its working 💯, GOOD LUCK in future projects 🎉🍀

**Review by rustam-tolipov — APPROVED** [review]
## Hi @mukhammadrizooff 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/display.js (hunk 1 above)
Comment (mukhammadrizooff) [reply src/display.js:11]: @Rustamxon7 Thank you!

**Pull request merged**